plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -89,7 +88,7 @@ export const CatalogPage: FC<{}> = () => {
         hidden: location ? location?.type !== 'github' : true,
```

Comment:
Use `location?.type !== 'github'` here as well? It's a lot cleaner.

Reply:
Done!